internal/config/config.go

```diff
@@ -213,7 +236,7 @@ func (c *AuthConfig) SetDefaultHost(host, source string) {
 // Login will set user, git protocol, and auth token for the given hostname.
 // If the encrypt option is specified it will first try to store the auth token
 // in encrypted storage and will fall back to the plain text config file.
-func (c *AuthConfig) Login(hostname, username, token, gitProtocol string, secureStorage bool) (bool, error) {
+func (c *AuthConfig) Login(hostname, username, token, protocol string, secureStorage bool) (bool, error) {
```

Comment:
Changed the argument name to not conflict with the new const `gitProtocol`.

Reply:
It might be useful in the future to suffix these constants like `gitProtocolSetting`, `gitProtocolConfig`, `gitProtocolKey` if need to avoid conflicts comes up again.

Reply:
Yeah this is a good call out, I will make that change.